### ISSUE TITLE
Healthcheck fractionals not supported

### ISSUE THREAD

**Issue by lindt** [hadolint/hadolint#893]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

```Dockerfile
HEALTHCHECK --interval=.1s --timeout=.1s --retries=100 CMD curl --fail -q localhost:80 || exit 1
```

is a valid statement in Dockerfile, but it fails.

### Actual behavior

the Healthcheck from above returns `unexpected '.' expecting positive number `

### Steps to reproduce the behavior

```bash
$ echo "HEALTHCHECK --interval=.1s --timeout=.1s --retries=100 CMD curl --fail -q localhost:80 || exit 1" | docker run -i hadolint/hadolint:2.10.0-beta hadolint -
```
